### PR TITLE
Feature meetings

### PR DESCRIPTION
**Pull request by capple2016**
Finally got the meeting model to work. There's no data filtering yet and I need to work on CSS stuff for create_meeting.html but hopefully this clears up any blocks in the road.

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -61,9 +61,6 @@
         <li class="nav-item">
           <a href="/meeting/projects" class="nav-link">Projects</a>
         </li>
-        <li class="nav-item">
-          <a href="/meeting/availability" class="nav-link">Availability</a>
-        </li>
```
Comment (calebpower) [inline templates/base.html:66]: Is Justin ready for this one?

**Review by calebpower — CHANGES_REQUESTED** [review]
I briefly looked through this, but and I saw a model change but I didn't see a change in the migrations file. I could have missed it, but was `python manage.py makemigrations` executed?

File: templates/base.html (hunk 1 above)
Comment (capple2016) [reply templates/base.html:66]: Not sure where this came into play, but it was not included in the new push.

**Comment by capple2016** [comment]
Will fix and reopen a pr.

**Comment by capple2016** [comment]
Added the migrations file for the meeting model.

**Review by calebpower — APPROVED** [review]

**Pull request merged**